### PR TITLE
Add SonarQube Analysis to code

### LINKED ISSUE #52: SonarQube Integration
**Is your feature request related to a problem? Please describe.**
We should use a tool to measure the QA of our code. 

**Describe the solution you'd like**
We should integrate in our CI/CD workflow the SonarQube to measure the QA of our project.

### PR DESCRIPTION
**Pull request by oidacra**
Add featured Github Action, every time make a push to master, run the new analysis.

fixes #52

### CODE REVIEW

**Comment by oidacra** [comment]
Need add the SONAR_TOKEN secret to the proyect.

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sonar.organization=test-angular-commmunity
+sonar.projectKey=oidacra_angular-communities
```
Comment (tonivj5) [inline sonar-project.properties:2]: is needed to change these properties?

Comment (oidacra) [reply sonar-project.properties:2]: yes, when you connect the project with github you have this 2 constants at the right-bottom of the project.
Screen Shot 2020-06-30 at 4 44 49 PM

**Comment by Caballerog** [comment]
Oh! Thanks! I just to configure SonarCloud to the voidcosmos organization account. I'm going to test the new workflow for sonarQube!

**Pull request merged**